2dcontext/imagebitmap/createImageBitmap-transfer.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>createImageBitmap transferring test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/media.js"></script>
+<script src="/common/namespaces.js"></script>
+<script src="common.sub.js"></script>
+<div id=log></div>
+<script>
+let worker, tests = {};
```

Comment:
Can you rename tests to continuations or something? It's a map from test names to callbacks, not what I thought at first.